packages/renderer/src/lib/feedback/feedbackForms/GitHubIssueFeedback.svelte

```diff
@@ -47,11 +52,11 @@ async function openGitHubIssues(): Promise<void> {
   </svelte:fragment>
   <svelte:fragment slot="validation">
     {#if !issueTitle || !issueDescription}
-      <ErrorMessage class="text-xs" error={issueValidaionError}/>
+      <ErrorMessage class="text-xs" error={issueValidationError}/>
     {/if}
   </svelte:fragment>
   <svelte:fragment slot="buttons">
-    <Button class="underline" type="link" aria-label="Cancel" on:click={() => onCloseForm()}>Cancel</Button>
-    <Button aria-label="Preview on GitHub" on:click={() => openGitHubIssues()} disabled={!issueTitle || !issueDescription}>Preview on GitHub</Button>
+    <Button class="underline" type="link" aria-label="Cancel" onclick={() => onCloseForm?.()}>Cancel</Button>
+    <Button aria-label="Preview on GitHub" onclick={() => openGitHubIssues()} disabled={!issueTitle || !issueDescription}>Preview on GitHub</Button>
```

Comment:
AFAIK you can't change on:click to onclick as the component is still expecting an event, not a callback